apis/core/v1beta1/schema/flags.json

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
Open question:

Should the schemas be added here as:

- url link
- local copy (as it is now)
- as a sub-repository

Reply:
+1 to have them as git submodels. This is how we do it many other repositories.

Reply:
Experiencing problems to initialize the submodule and adding it to the built image, do you have an idea how this should be done properly? I am unable to make it work.

Reply:
ok, adapted!